modules/python/src/custom_nodes/google_genmedia/utils.py

```diff
@@ -752,86 +562,19 @@ def generate_video_from_text(
     config = GenerateVideosConfig(**temp_config)
     print(f"Config for text-to-video generation: {config}")
 
-    retries = 0
-    while retries <= retry_count:
-        try:
-            print("Sending request to Veo API for text-to-video generation...")
-            operation = client.models.generate_videos(
-                model=model, prompt=prompt, config=config
-            )
-            print(f"Initial operation response object type: {type(operation)}")
-
-            operation_count = 0
-            while not operation.done:
-                time.sleep(20)  # Polling interval
-                operation = client.operations.get(operation)
-                operation_count += 1
-                print(f"Polling operation (attempt {operation_count})...")
-
-            print(f"Operation completed with status: {operation.done}")
-            return process_video_response(operation)
-
-        except genai_errors.ClientError as e:
-            if e.code == StatusCode.RESOURCE_EXHAUSTED:
-                retries += 1
-                if retries <= retry_count:
-                    retry_wait = retry_delay
-                    print(
-                        f"API Quota/Resource Exhausted (attempt {retries}/{retry_count}) - "
-                        f"Code: {e.code.name}. Waiting {retry_wait:.1f} seconds before retry. Error: {e.details}"
-                    )
-                    time.sleep(retry_wait)
-                else:
-                    raise RuntimeError(
-                        f"API Quota/Resource Exhausted after {retries} attempts for {model} (Code: {e.code.name}). "
-                    )
-            elif e.code == StatusCode.INVALID_ARGUMENT:
-                raise ValueError(
-                    f"Invalid API argument supplied. Check your prompt and parameters. Error: {e.details}"
-                )
-            elif (
-                e.code == StatusCode.PERMISSION_DENIED
-                or e.code == StatusCode.UNAUTHENTICATED
-                or e.code == StatusCode.FORBIDDEN
-            ):
-                raise RuntimeError(
-                    f"Permission denied. Check your GCP service account permissions for Veo API. Error: {e.details}"
-                )
-            else:
-                # Catch any other ClientError that's not specifically handled (e.g., Bad Request, Conflict)
-                raise RuntimeError(
-                    f"Unexpected Veo API Client Error (Code: {e.code.name}). Error: {e.details}"
-                )
+    print("Sending request to Veo API for text-to-video generation...")
+    operation = client.models.generate_videos(model=model, prompt=prompt, config=config)
```

Comment:
this should be in a try except block to catch the cascading exception and then print it and exit the program.